recipes/server.rb

```diff
@@ -22,4 +22,8 @@
 include_recipe 'lvm'
 include_recipe 'gluster::server_setup'
 include_recipe 'gluster::server_extend'
-include_recipe 'gluster::volume_extend'
+include_recipe 'gluster::volume_extend' if begin
+                                             Gem::Specification.find_by_name('di-ruby-lvm')
+                                           rescue Gem::LoadError
+                                             puts 'di-ruby-lvm not found'
```

Comment:
Also, the message should be a little more descriptive
Like: `not including gluster::volume_extend since di-ruby-lvm was not found`